Logger/FileLogger.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public String filePath { get; }
+
+        internal FileLogger(String filePath)
+        {
+            this.filePath = filePath;
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            if (message is null)
+            {
+                throw new ArgumentNullException("Null message");
+            }
+            StringBuilder sb = new StringBuilder();
+            sb.Append(System.DateTime.Now + " ");
+            sb.Append(this.name + " ");
+            sb.Append(logLevel.ToString() + " ");
+            sb.Append(message + Environment.NewLine);
```

Comment:
This works, but there is also a convenience method on StringBuilder that will append the new line for you.
```suggestion
            sb.AppendLine(message);
```